inspirehep/modules/workflows/workflows/article.py

```diff
@@ -376,7 +380,8 @@ class Article(object):
                         # TODO: once legacy is out, this should become
                         # unconditional, and remove the SEND_TO_LEGACY_AND_WAIT
                         # steps
```

Comment:
However the call to `SEND_TO_LEGACY_AND_WAIT` should be amended to always send the record in `replace` mode: